bibtex-actions.el

```diff
@@ -132,8 +133,14 @@ you can use the following initial inputs: \"has:pdf\",
   :group 'bibtex-actions
   :type '(alist :key-type symbol
                 :value-type (choice string
-                                    (const :tag "No initial input" nil))))
-  
+                                    (const :tag "No initial input" nil)
+                                    (const :tag "Key at point" bibtex-completion-key-at-point))))
+
+(defcustom bibtex-actions-default-action 'bibtex-actions-open
```

Comment:
Should this name be specific to "open" actions?

Reply:
It is used by `bibtex-actions-at-point`, which could include other actions such as "copy", "search", etc

It is better not to rename it unless you want to remove or rename `bibtex-actions-at-point`